src/uwuw/uwuw_preprocessor.cpp

```diff
@@ -152,7 +152,7 @@ void uwuw_preprocessor::process_materials() {
 
     std::string mat_name = dmd->return_property(*s_it, "mat");
     std::string density  = dmd->return_property(*s_it, "rho");
-
+    mat_name = "mat:"+mat_name;
```

Comment:
This seems like a big difference, what am I missing here?

Looks like ASTYLE fail here too 🙂

Reply:
Yikes. That was a personal patch, how embarrassing... :grimacing: will remove :smile: